frontend/src/Explorer/Util/Factory.purs

```diff
@@ -20,10 +24,16 @@ mkCoin coin =
 mkCAddress :: String -> CAddress
 mkCAddress = CAddress
 
+-- | Helper to summarize coins by a list of Tx inputs or outputs
+sumCoinOfInputsOutputs :: Array (Tuple CAddress Coin) -> Coin
```

Comment:
This looks a bit verbose to me, it could be just me. I prefer (if that was the intention):
```purescript
-- | Helper to summarize coins by a list of Tx inputs or outputs
sumCoinOfInputsOutputs :: Array (Tuple CAddress Coin) -> Coin
sumCoinOfInputsOutputs list =
    mkCoin $ sum $ addressCoins <$> list
      where
        -- | Get total number of coins from an address
        addressCoins :: (Tuple CAddress Coin) -> Int
        addressCoins (Tuple _ coin) = coin ^. (_Coin <<< getCoin)
```